source/supported-hardware/up2.rst

```diff
@@ -21,6 +21,7 @@ Board Setup
 ^^^^^^^^^^^^^^^^^
 
 .. image:: /images/up2_setup.jpg
+   :width: 600

```

Comment:
Is this applicable for up2 page only? if not consistent with all other pages, better to keep it as others..

Reply:
only two images in the documentation - fixed both.